juju/charmhub.py

```diff
@@ -41,6 +41,16 @@ async def is_subordinate(self, charm_name):
     # TODO (caner) : we should be able to recreate the channel-map through the
```

Comment:
Does this TODO apply to list_resources or info?

Reply:
All of it, really, since if we create a channel-map dict then all of these methods are gonna change, maybe moving this to the top would be more appropriate